src/containers/Header/Search.tsx

```diff
@@ -45,7 +46,13 @@ const determineHashType = async (id: string, rippledContext: XrplClient) => {
     await getTransaction(rippledContext, id)
     return 'transactions'
   } catch (e) {
-    return 'nft'
+    try {
+      await getLedgerEntry(rippledContext, id)
+      return 'entry'
```

Comment:
Is there any way we can cache the result of this method invocation? It feels wasteful to invoke this API only for the purpose of checking if the hash pertains to a transaction, ledger-entry (or) an NFT.